README.md

```diff
@@ -1,9 +1,213 @@
+the `Installation manual` section in the [documentation](#documentation) chapter of this document.
+
+# Configuration
+
+The *Integration Service* can be configured during runtime by means of a dedicated **YAML** file.
+This YAML configuration file must follow a specific syntax, this meaning that several sections
+are required for it to sucessfully configure and launch an *Integration Service* instance;
+while others are optional. Both of them are listed and reviewed here:
+
+* `types` *(optional)*: It allows to list the [IDL](https://www.omg.org/spec/IDL/4.2/About-IDL/)
+  types used by the *Integration Service* to later define the topics and services types which will
+  take part in the communication process.
+
+  This field can be omitted for certain *Integration Service* instances where one or more *System
+  Handles* already include static type definitions and its corresponding transformation libraries
+  (*Middleware Interface Extenion* or *mix* files).
+
+  ```yaml
+    types:
+      idls:
+        - >
+          #include <GoodbyeWorld.idl>
+          struct HelloWorld
+          {
+            string data;
+            GoodbyeWorld bye;
+          };
+      paths: [ "/home/idl_files/goodbyeworld/" ]
+  ```
+
+  <details>
+  <summary>Several parameters can be configured within this section: <i>(click to expand)</i></summary>
+
+    * `idls`: List of IDL type definitions which can be directly embedded within the configuration file. If the *types* section is defined, this subsection is mandatory. The type can be entirely defined within the YAML file, or can be included from a preexisting IDL file; for the latter, the system path containing where the IDL file is stored must be placed into the `paths` section.
+
+    * `paths` *(optional):* Using this parameter, an existing IDL type written in a separate file can be included within the *Integration Service* types section. If the IDL path is not listed here, the `#include` preprocessor directive will fail.
+
+  </details>
+
+* `systems`: Specifies which middlewares will be involved in the communication process, allowing
+  to configure them individually.
+
+  Some configuration parameters are common for all of the supported middlewares within the
+  *Integration Service* ecosystem; while others are specific of each middleware. To see which
+  parameters can be tuned for a certain middleware, please refer to its dedicated *README* section
+  in its corresponding GitHub repository, under the name of `https://github.com/eProsima/<MW_NAME>-SH`.
+
+  ```yaml
+    systems:
+      foo: { type: foo }
+      bar: { type: bar, types-from: foo }
+  ```
+
+  <details>
+  <summary>In relation to the common parameters, their behaviour is explained in the following section: <i>(click to expand)</i></summary>
+
+    * `type`: Middleware or protocol kind. Up to this time, supported middlewares are: *Fast DDS*, *ROS1*, *ROS2* and *WebSocket*.
```

Comment:
This sentence is a bit confusing, because you say that in the type you have to specify the middleware and which ones we support, but that is not the syntax to use.